Libraries/HeapCapture/NativeHeapCapture.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import {Platform} from 'react-native';
+
+export interface Spec extends TurboModule {
+  // Common interface
+  +captureHeap: (path: string) => void;
+
+  // Android only
```

Comment:
nit: It looks like this module is only available on Android. So, this comment is confusing. Should it be here?